deploy-manage/users-roles/cluster-or-deployment-auth/native.md

```diff
@@ -4,24 +4,100 @@ mapped_urls:
+You can configure a `native` realm in the `xpack.security.authc.realms.native` namespace in `elasticsearch.yml`. Explicitly configuring a native realm enables you to set the order in which it appears in the realm chain, temporarily disable the realm, and control its cache options.
 
-$$$managing-native-users$$$
\ No newline at end of file
+1. Add a realm configuration to `elasticsearch.yml` under the `xpack.security.authc.realms.native` namespace. It is recommended that you explicitly set the `order` attribute for the realm.
+
+    ::::{note} 
+    You can configure only one native realm on {{es}} nodes.
+    ::::
+
+
+    See [Native realm settings](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#ref-native-settings) for all of the options you can set for the `native` realm. For example, the following snippet shows a `native` realm configuration that sets the `order` to zero so the realm is checked first:
+
+    ```yaml
+    xpack.security.authc.realms.native.native1:
+      order: 0
+    ```
+
+    ::::{note} 
+    To limit exposure to credential theft and mitigate credential compromise, the native realm stores passwords and caches user credentials according to security best practices. By default, a hashed version of user credentials is stored in memory, using a salted `sha-256` hash algorithm and a hashed version of passwords is stored on disk salted and hashed with the `bcrypt` hash algorithm. To use different hash algorithms, see [User cache and password hash algorithms](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#hashing-settings).
+    ::::
+
+2. Restart {{es}}.
+
+
+## Manage native users using {{kib}} [managing-native-users]
+
+The Elastic enables you to easily manage users in {{kib}} on the **Stack Management > Security > Users** page. From this page, you can create users, edit users, assign roles to users, and change user passwords. You can also deactivate or delete existing users.
+
+### Example: Create a user [_create_a_user]
+
+1. Navigate to **Stack Management**, and under **Security**, select **Users**.
+2. Click **Create user**.
+3. Give the user a descriptive username, and choose a secure password.
+4. Optional: assign [roles](/deploy-manage/users-roles/user-roles.md) to the user.
+5. Click **Create user**.
+
+:::{image} ../../../images/kibana-tutorial-secure-access-example-1-user.png
+:alt: Create user UI
+:class: screenshot
+:::
+
+## Manage native users using the `user` API
+
+You can manage users through the Elasticsearch `user` API. 
+
+For example, you can change a user's password:
+
+```console
+POST /_security/user/user1/_password
+{
+  "password" : "new-test-password"
+}
+```
+
+For more information and examples, see [Users](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-security).
+
+## Reset passwords for native users using the command line [reset-pw-cmd-line]
+
+:::{applies}
+:stack: all
+:::
```

Comment:
does this work for k8s or others? there's a note about these utils working on kubernetes here
https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-users-and-roles.html#k8s_creating_custom_users